src/uct/sm/mm/mm_posix.c

```diff
@@ -17,12 +17,16 @@
 #define UCT_MM_POSIX_MMAP_PROT      (PROT_READ | PROT_WRITE)
 #define UCT_MM_POSIX_HUGETLB        UCS_BIT(0)
 #define UCT_MM_POSIX_SHM_OPEN       UCS_BIT(1)
-#define UCT_MM_POSIX_CTRL_BITS      2
+#define UCT_MM_POSIX_PROC_LINK      UCS_BIT(2)
+#define UCT_MM_POSIX_CTRL_BITS      3
+#define UCT_MM_POSIX_FD_BITS        29
```

Comment:
@alinask I do not think we should assume that is always the case. If there are large amount of files opened in the system concurrently, the value of fd is possible to be large. Adding an assert should be a proper solution for now. @yosefe what do you think?